src/workerd/api/web-socket.c++

```diff
@@ -12,6 +12,30 @@
 
 namespace workerd::api {
 
+bool WebSocketProtocolError::decodeFromException(const kj::Exception& ex) {
```

Comment:
Non-blocking nit: it would be helpful to add a comment in here indicating what a true or false response from this method means.

Reply:
I wound up reworking this a bit so it's harder to misuse; instead of returning true when we found something and relying on the caller to not use the WebSocketProtocolError object otherwise, I dropped this method in favor of a static `fromException` method that returns `kj::Maybe<WebSocketProtocolError>`, hopefully making it harder to misuse.